wp-content/themes/reconasia/assets/_scss/pages/404.scss

```
@@ -0,0 +1,40 @@
+@use '../abstracts' as *;
+
+@use '../components/entry-header';
+
+.error-404 {
+  display: flex;
+  justify-content: center;
+  margin-top: rem(40);
+  margin-bottom: rem(56);
+
+  @include breakpoint('medium') {
+    margin-top: rem(56);
+  }
+
+  &__content {
+    max-width: $size__content-max-width;
+  }
+
+  &__title {
+    color: $color-link-primary-200;
+    @extend %text-style-heading-3-x-large-strong;
```

This should be `%text-style-label-3-x-large`
```suggestion
    @extend %text-style-label-3-x-large;
```